warehouse/accounts/views.py

```diff
@@ -372,6 +376,9 @@ def webauthn_authentication_validate(request):
             .hexdigest()
             .lower(),
         )
+
+        send_recovery_code_reminder_email(request, request.user)
```

Comment:
I don't see how this is conditional on _not_ having recovery codes... seems like it'd fire regardless?